src/bot/checks.rs

```diff
@@ -0,0 +1,137 @@
+use poise::serenity_prelude::{ChannelId, Permissions, RoleId};
+use sqlx::{MySql, Pool};
+use std::env;
+use tracing::error;
+
+use crate::bot::Context;
+use crate::mysql_lib;
+
+/// Checks if the author has any of the specified roles
+///
+/// Will return false when:
+/// - the message was not send in a guild
```

Comment:
Doesn't seem to be implemented, is this implied by something else?

Reply:
`ctx.author_member()` will return `None` if the message is send outside of a guild, thus returning false